simpledotorg/simple-server
# Add diabetes registration and follow up graph

**Story card:** [sc-7780](https://app.shortcut.com/simpledotorg/story/7780/add-registrations-and-follow-ups-graph)

## Because

We want to show cumulative and monthly diabetes registrations and follow ups in the dashboard

## This addresses

- Region Summary and Region Summary Schema report cumulative registrations, monthly registrations and monthly follow ups
- Repository and Repository presenter present data to the controller
- The diabetes view adds html to display the Registrations and follow up diabetes graph

## Test instructions

- Turn on diabetes_management_reports on flipper
- Visit the reports page of a facility which has enable_diabetes_management set to true
- Click on Diabetes link
- You should see the Registrations and follow-ups graph

![image](https://user-images.githubusercontent.com/3933133/166631377-d3312c82-0137-454e-b360-840536af3b7e.png)
